src/components/Personalization/index.js

```diff
@@ -11,13 +11,17 @@ governing permissions and limitations under the License.
 */
 
 import { isNonEmptyArray, groupBy } from "../../utils";
-import { string } from "../../utils/validation";
+import { string, boolean, arrayOf, objectOf } from "../../utils/validation";
 import { initRuleComponentModules, executeRules } from "./turbine";
 import { hideContainers, showContainers } from "./flicker";
 import collectClicks from "./helper/clicks/collectClicks";
 
 const DECISIONS_HANDLE = "personalization:decisions";
 const PAGE_WIDE_SCOPE = "page_wide_scope";
+const GET_DECISIONS_OPTIONS_SCHEMA = {
+  viewStart: boolean().default(null),
```

Comment:
Could you explain why `viewStart` is `null` by default instead of `false`?

Reply:
My bad, I was thinking to make a difference when it is passed or not. but making it `false` by default is a better option.